backend/src/main/java/org/cryptomator/hub/license/LicenseHolder.java

```diff
@@ -67,6 +67,17 @@ public DecodedJWT get() {
 		return license;
 	}
 
+	/**
+	 * Checks if the license is valid.
+	 *
+	 * @return {@code true}, if the license _is set and valid_. Otherwise false.
+	 */
+	public boolean isValid() {
```

Comment:
bad name, since "valid" is too broad for just checking its validity period. How about negating the result and calling it `isExpired`?

Reply:
Why not splitting it into two functions then? A tad more code, but the meaning is clear.
* `boolean hasLicense()`
* `boolean isExpired()`

Reply:
Ended up with `isSet()`, see 325d050656758bd0659785e74fa4d4949a9ce85b.